Library/Homebrew/cask/config.rb

```diff
@@ -30,13 +30,13 @@ class Config
         vst3_plugindir:       "~/Library/Audio/Plug-Ins/VST3",
         screen_saverdir:      "~/Library/Screen Savers",
       }.freeze,
-      T::Hash[Symbol, String],
+      T::Hash[Symbol, T.nilable(String)],
     )
 
     sig { returns(T::Hash[Symbol, T.untyped]) }
```

Comment:
Could we tighten these up, as [below](Library/Homebrew/extend/os/linux/cask/config.rb)?

```suggestion
      T::Hash[Symbol, String],
    )

    sig { returns(T::Hash[Symbol, String]) }
```